plugins/txt/plugin/commands/txt_new_scene.rb

```diff
@@ -0,0 +1,66 @@
+module AresMUSH
+  module Txt
+      class TxtNewSceneCmd
+        include CommandHandler
+
+        attr_accessor :names, :names_raw, :message, :scene_id
+
+    def parse_args
+      if (!cmd.args)
+        self.names = []
+      else
+        args = cmd.parse_args(ArgParser.arg1_equals_arg2)
+        if (args.arg1 && (args.arg1.include?("http://") || args.arg1.include?("https://")) )
+          self.names = []
+        end
+        self.names = list_arg(args.arg1)
+        self.names_raw = trim_arg(args.arg1)
+        self.message = trim_arg(args.arg2)
+      end
+    end
+
+    def check_txt_target
+      return t('txt.txt_new_scene_target_missing') if !self.names || self.names.empty?
+      return nil
+    end
+
+    def handle
+
+      self.names.each do |name|
+        result = Character.named(name)
+        if !result
+          client.emit_failure t('txt.no_such_character')
+          return
+        end
+      end
+
+      scene = Scene.create(owner: enactor,
```

Comment:
Suggest using the `Scenes.start_scene` helper here.  Right now this is doing everything that start-scene does, but if I ever add additional functionality that needs to happen when a scene is started, this'll miss it.

Reply:
Changed.